spring-cloud-gcp-data-spanner/src/test/java/org/springframework/cloud/gcp/data/spanner/core/admin/SpannerSchemaUtilsTests.java

```diff
@@ -64,10 +65,10 @@ public void getDropDDLTest() {
 
 	@Test
 	public void getCreateDDLTest() {
-		assertEquals("CREATE TABLE custom_test_table ( id STRING(MAX) , id2 INT64 "
-				+ ", custom_col STRING(MAX) , other STRING(333) , bytes BYTES(MAX) "
-				+ ", bytesList ARRAY<BYTES(111)> , integerList ARRAY<INT64> "
-				+ ", doubles ARRAY<FLOAT64> ) PRIMARY KEY ( id , id2 )",
+		assertEquals("CREATE TABLE custom_test_table ( id STRING(MAX) , id3 INT64 , "
+						+ "id2 STRING(MAX) , bytes2 BYTES(MAX) , custom_col STRING(MAX) , other STRING(333) ,"
```

Comment:
id3 has (keyOrder = 3), while id2 has (keyOrder = 2)
why does id2 appears after id3?

Reply:
I wanted to write the TestEntity class with the keys out of order to make sure that we're not depending on that written order, but only on the key order. 

As a side effect in the SQL it comes out in the same reversed order. However, what actually matters is that final line where the order does matter, and it is correct there: `PRIMARY KEY ( id , id2 , id3 )`

Reply:
ok I see